analyses/telomerase-activity-prediction/01-run-EXTEND.R

```diff
@@ -8,6 +8,7 @@ suppressPackageStartupMessages({
   library(optparse)
 })
 stringsAsFactors=FALSE
+`%>%` <- dplyr::`%>%`
```

Comment:
I assume this got added because of `filter_expression_by_tumor_purity()`. From a design perspective, tucking this into the function itself might be preferable.

Reply:
Done in https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1657/commits/15b058b61eaacebfa1ffa6b1ddc9f2655c1bc267